build.assets/windows/build.ps1

```diff
@@ -199,4 +199,26 @@ function Convert-Base64 {
         $bytes = [Convert]::FromBase64String($Data)
         Set-Content -Encoding Byte -Path $FilePath -Value $bytes
     }
+}
+
+function Send-ErrorMessage {
+    <#
+    .SYNOPSIS
+    Formats and sends a buidl failure message to Slack
+    #>
+    [CmdletBinding()]
+    param ()
+
+    begin {
+        $BuildUrl = "$Env:DRONE_SYSTEM_PROTO`://$Env:DRONE_SYSTEM_HOSTNAME/$Env:DRONE_REPO_OWNER/$Env:DRONE_REPO_NAME/$Env:DRONE_BUILD_NUMBER"
+        $GoOS = $(go env GOOS)
+		$GoArch = $(go env GOARCH)
+        $Msg = @"
+THIS IS A TEST (by @trent): Warning: ``$GoOS-$GoArch`` artifact build failed for [``$Env:DRONE_REPO_NAME``] - please investigate immediately!
```

Comment:
Do we want to remove the "THIS IS A TEST" message now this works :)

Reply:
https://www.youtube.com/watch?v=wdpLWML_tDU